boards/nxp/frdm_mcxc242/doc/index.rst

```diff
@@ -0,0 +1,221 @@
+.. _frdm_mcxc242:
+
+NXP FRDM-MCXC242
+################
+
+Overview
+********
+
+FRDM-MCXC242 is a compact and scalable development board for rapid
+prototyping of MCX C242 MCU. It offers industry standard headers
+for easy access to the MCU’s I/Os, integrated open-standard serial
+interfaces and on-board MCU-Link debugger.
+The MCXC is a general purpose ultra-low-power MCU family,
+providing additional memory, communications and analog peripheral.
+
+
+.. image:: frdm_mcxc242.webp
+   :align: center
+   :alt: FRDM-MCXC242
+
+Hardware
+********
+
+- MCXC242VLH Arm Cortex-M0+ microcontroller running at 48 MHz
+- 64LQFP package
+- 64KB flash
+- 16KB SRAM
+- USB FS 2.0
+- 2x low-power UART, 1x UART, 2x I2C, 2x SPI
+- FXLS8974CF accelerometer
+- NTC temperature sensor
+- RGB user LED
+- On-board MCU-Link debugger with CMSIS-DAP
+- Arduino Header, mikroBUS, Pmod
+
+For more information about the MCXC242 SoC and FRDM-MCXC242 board, see
+these references:
+
+- `MCX C14x/24x/44x Website`_
+- `MCX C24x Datasheet`_
+- `MCX C24x Reference Manual`_
+- `FRDM-MCXC242 Website`_
+- `FRDM-MCXC242 User Manual`_
+- `FRDM-MCXC242 Design Files`_
+
+Supported Features
+==================
+
+The FRDM-MCXC242 board configuration supports the following hardware features:
```

Comment:
use west board name with \`\` around it and *board target

Reply:
Fixed.